db.js

```diff
@@ -308,6 +308,28 @@ var loans = (function () {
       return loanData;
     },
 
+    insert: function (loan) {
+      var sql =
+        "INSERT INTO loans (requester_id, book_id, owner_contact, requester_contact, " +
+        "request_date, accept_date, loan_start_date, loan_end_date, return_date, status) " +
+        "VALUES (?,?,?,?,?,?,?,?,?,?)";
+      var inserts = [
+        loan.requester_id,
+        loan.book_id,
+        loan.owner_contact,
+        loan.requester_contact,
+        loan.request_date,
```

Comment:
Request date should be set to something like `Date.now()`. I'm not actually passing in a date on the frontend, so this value will be undefined. Currently, the only values that will be defined here are `requester_id`, `book_id`, and `requester_contact` since those are what the frontend sends.

Reply:
Hm. Well, when does the `owner_contact` get set then? The mappers expect this field to be not empty.

Reply:
Tait and I just discussed that. We're thinking of just setting it as an empty string and moving into a more detailed discussion of it next week.